Aspire BudgetingTests/GoogleSheetsValidatorTests.swift

```diff
@@ -0,0 +1,112 @@
+//
+// GoogleSheetsValidatorTests.swift
+// Aspire BudgetingTests
+//
+
+@testable import Aspire_Budgeting
+import Combine
+import GoogleAPIClientForREST
+import XCTest
+
+class GoogleSheetsValidatorTests: XCTestCase {
+
+  var validatorSink: AnyCancellable?
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/c832bf4daa051d3d91d7e9434f5fff223299bc47/Aspire BudgetingTests/GoogleSheetsValidatorTests.swift#L20"></span>Line should be 100 characters or less: currently 111 characters<br />
<code>line_length</code> <code>GoogleSheetsValidatorTests.swift:20</code></td>
    </tr>
  </tbody>
</table>